setup.py

```diff
@@ -26,7 +25,9 @@
             "db-queries",
             "gbd",
             "elmo",
-            "ihme-rules"],
+            "ihme-rules",
+            "jobmon==0.0.5"
```

Comment:
Thanks @mbannick ! Let me know if this is good :)

Reply:
yes!! looks great!